parser/src/ast/mod.rs

```diff
@@ -14,9 +20,14 @@ pub use transition_constraints::*;
 pub struct Source(pub Vec<SourceSection>);
 
 /// Source is divided into SourceSections.
-/// There are 4 types of Source Sections:
+/// There are 6 types of Source Sections:
 /// - AirDef: Name of the air constraints module.
 /// - TraceCols: Trace Columns representing columns of the execution trace.
+/// - PublicInputs: Public inputs are each represented by a fixed-size array. They are not required,
+///   and there is no limit to the number of public inputs that can be specified.
```

Comment:
This could be done in the future PRs: public inputs should actually be required. An AIR without public inputs is kind of meaningless.

Reply:
I've opened #30